### DISCUSSION TITLE
Help: ChatTTS: 'Chat' object has no attribute 'speaker'

### DISCUSSION THREAD

**Discussion by sheepying** [langgenius/dify#9712] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I encountered "**InvokeBadRequestError: Failed to speech the text, detail: [address=0.0.0.0:35231, pid=44541] 'Chat' object has no attribute 'speaker'** "when calling ChatTTS.

The model is deployed by Xinference.  

The download source for my ChatTTS model is: **git clone https://www.modelscope.cn/pzc163/chatTTS.git**

### 2. Additional context or comments

The complete error message is as follows,

![屏幕截图 2024-10-23 170414](https://github.com/user-attachments/assets/bac3e287-ba84-4839-93f5-da2f2bd0c27a)

**Comment by nandishaivalli** [comment]
I'm gettign the same error did you find any solutions
